_docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhd-frontend/src/scripts/@rhd/rhdp-search/rhdp-search-app.ts

```diff
@@ -75,86 +69,88 @@ export default class RHDPSearchApp extends HTMLElement {
     sort = new RHDPSearchSortPage();
 
     filterObj = {
```

Comment:
Reordering the data in this json object affects the load order of filters in the left rail.